data/crac-file/crac-api/src/main/java/com/farao_community/farao/data/crac_api/SynchronizationException.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_api;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class SynchronizationException extends Exception {
```

Comment:
Could we use FaraoException in commons to be consistent with other part of code? or SynchronizationException has something different from FaraoException, then it would be good to note it in the header comment.

Reply:
I added a comment